api/util/validationSchemas.js

```diff
@@ -31,9 +31,9 @@ exports.incidentObject = yup.object({
   force_rank: yup.string().required().trim().min(2).max(50),
   status: yup
     .string()
-    .required()
```

Comment:
wouldn't this be required since it would default to pending?

Reply:
If it were required then it would have to be passed in the request body every time.
After the middleware, the model function will set status to what the admin defines or 'approved' by default because the admin is posting it themselves

Reply:
Makes sense, might be worthwhile to discuss data flow in the future.